common/src/main/java/com/microsoft/identity/common/internal/fido/PublicKeyCredentialDescriptor.kt

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.fido
+
+/**
+ * Representation of WebAuthn's PublicKeyCredentialDescriptor.
+ * https://w3c.github.io/webauthn/#dictdef-publickeycredentialdescriptor
+ */
+@kotlinx.serialization.Serializable
+data class PublicKeyCredentialDescriptor(
+    val type: String,
+    val id: String
+)
```

Comment:
nit: empty line

Reply:
Checking all the new files in this PR, I think I do already have an empty line at the end of each. Unless I'm misunderstanding the comment here.